### PR TITLE
[quest] ADDED: 'Water Shield' Shaman Fake Rune Quests To QuestDB

### LINKED ISSUE #5606: [quest] 'Water Shield' Shaman Rune Quest Missing From QuestDB
## Description

The SoD Shaman rune quest 'Water Shield' is not currently in the QuestDB. 

## Wowhead link

https://www.wowhead.com/classic/guide/season-of-discovery/shaman-class-runes-discovery-location
https://www.wowhead.com/classic/spell=410097/engrave-gloves-water-shield

## Screenshots

N/a

## Questie & WoW version

v9.2.17
WoW Classic SoD - v1.15.052409

### PR DESCRIPTION
**Pull request by Sommos**
## Issue references

Fixes #5606
Linked To #5443

## Proposed changes

- ADDED: All 'Water Shield' Shaman fake rune quests and they are now in the QuestDB. The Silverpine Forest location is accessible to users, however The Barrens location is not currently due to Kolkar Centaur spawns.

### CODE REVIEW

File: Database/Corrections/sodQuestFixes.lua [hunk 1]
```diff
@@ -3168,6 +3168,30 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410107,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90214] = {
+            [questKeys.name] = "Water Shield",
+            [questKeys.startedBy] = {{3272,3273,3394},{3642}},
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:3173]: For the "Regeneration" and "Savage Roar" rune, we (aka me) did not add the NPCs to receive the key 🤔 But we (me again) did also not add a description how to receive the key in the objectivesText.

I say we remove the NPCs here as well, but keep description and also add such a description for the runes listed above.

Moreover `152608` and `152618` are also valid Kolkar Booties.

```suggestion
            [questKeys.startedBy] = {nil,{152608,152618,3642}},
```

File: Database/Corrections/SeasonOfDiscovery.lua [hunk 2]
```diff
@@ -361,6 +363,7 @@ local questsToBlacklistBySoDPhase = {
         [90200] = true, -- Hiding Rogue Mutilate Dun Morogh for now as there are too many icons
         [90201] = true, -- Hiding Rogue Shiv Duskwood for now as there are too many icons
         [90204] = true, -- Hiding Shaman Way of Earth Silverpine Forest for now as there are too many icons
+        [90214] = true, -- Hiding Shaman Water Shield The Barrens for now as there are too many icons
```
Comment (BreakBB) [inline Database/Corrections/SeasonOfDiscovery.lua:366]: If we only show the chest locations we do not need to hide this rune.

```suggestion
```

**Review by BreakBB — CHANGES_REQUESTED** [review]

File: Database/Corrections/SeasonOfDiscovery.lua (hunk 2 above)
Comment (Sommos) [reply Database/Corrections/SeasonOfDiscovery.lua]: Fixed in: https://github.com/Questie/Questie/pull/5610/commits/9df6737c3ece0b93e05f8449b1800e640518c8ee

File: Database/Corrections/sodQuestFixes.lua (hunk 1 above)
Comment (Sommos) [reply Database/Corrections/sodQuestFixes.lua]: Fixed in: https://github.com/Questie/Questie/pull/5610/commits/9df6737c3ece0b93e05f8449b1800e640518c8ee

**Review by BreakBB — APPROVED** [review]

**Pull request merged**